cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/NvaProject.java

```diff
@@ -373,6 +376,14 @@ public void setCreator(NvaContributor creator) {
         this.creator = creator;
     }
 
+    public URI getWebPage() {
```

Comment:
This may be null, sometimes I wonder if Optional is a good return type to make this clear…possibly not in this case.